api/src/templates/login/token-login.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>{{ branding.name }}</title>
+  <link href="/login/style.css" media="all" rel="stylesheet">
+  <link rel="shortcut icon" href="/favicon.ico">
+</head>
+<body data-default-locale="{{ defaultLocale }}" data-translations="{{ translations }}">
+<div class="center">
+  <div id="wrapper">
+    <img class="logo" src="{{ branding.logo }}" title="{{ branding.name }}">
+    <div class="locale-wrapper">
+      <div class="loading">
+        <p translate="login.token.loading"></p>
+        <div class="loader"></div>
+      </div>
+      <div class="errors">
+        <p class="error invalid" translate="login.token.missing.expired.invalid"></p>
+        <p class="error unknown" translate="login.token.general.error"></p>
+        <p translate="login.token.redirect.login.info"></p>
+        <a href="/medic/login" class="btn"><span translate="login.token.redirect.login"></span></a>
+      </div>
+    </div>
+    <div id="locale" class="locale-wrapper">
+      <% locales.forEach(function(locale) { %>
+      <a href="#" name="{{ locale.key }}" class="locale">{{ locale.label }}</a>
+      <% }); %>
+    </div>
+  </div>
+</div>
+<form id="tokenLogin" action="{{tokenUrl}}"></form>
+<script src="/login/script.js"></script>
+</body>
+</html>
```

Comment:
Looks like this (please don't mind my awesome branding logo!)

![image](https://user-images.githubusercontent.com/35681649/86963385-1c61f800-c16d-11ea-918f-6d134ce6929f.png)

![image](https://user-images.githubusercontent.com/35681649/86963422-2dab0480-c16d-11ea-89d1-bf8e4e93b03a.png)


![image](https://user-images.githubusercontent.com/35681649/86963433-34d21280-c16d-11ea-9483-9bc2207d88c8.png)

Reply:
It'd be good to add some additional error modes. For example we can give a different message for "timeout" vs "server error" vs "missing params" vs "token expiry". This would help remote debugging of login issues.

Reply:
Done!